libs/core/routing/src/uri/truncation/query-fragment.ts

```diff
@@ -1,6 +1,7 @@
 /**
  * Truncates query parameters and fragments from a URI.
  * e.g. /foo/bar/baz.html?thing=test#id -> /foo/bar/baz.html
+ * Note that this will not work as expected unless `uri` has a leading slash.
  */
 export const daffUriTruncateQueryFragment = (uri: string): string =>
```

Comment:
Rather than add a comment, can we handle the case by optionally adding a slash if the first char isn't a slash?

Reply:
yeah I think its better that way, changed